habitat_baselines/config/default.py

```diff
@@ -22,13 +22,51 @@
 # BASELINE
 # -----------------------------------------------------------------------------
 _C.BASELINE = CN()
+_C.BASELINE.TRAINER_NAME = "ppo"
```

Comment:
```suggestion
_C.TRAINER.TYPE = "ppo"
```
Let's be consistent with habitat-api naming: https://github.com/facebookresearch/habitat-api/blob/master/habitat/config/default.py#L61. Maybe, even `BASELINE` -> `TRAINER`.

Reply:
> Maybe, even BASELINE -> TRAINER

This is definitely something I would like to discuss. I'm thinking something like `BASELINE` -> `EXPERIMENT`.